package/utils/util-linux/Makefile

```diff
@@ -172,7 +172,7 @@ endef
 define Package/fdisk
 $(call Package/util-linux/Default)
   TITLE:=manipulate disk partition table
-  DEPENDS:= +libblkid +libsmartcols +libfdisk
+  DEPENDS:= +libblkid +libsmartcols +libfdisk +libncursesw
```

Comment:
We don't really have the manpower to maintain custom patches to left and right. Looking at most outdated packages those carry custom patches and I highly doubt it's worth it. Also, there's no libncursesw.so for now ;-)